roles/postgres/tasks/main.yml

```diff
@@ -61,7 +67,7 @@
   ansible.builtin.include_tasks:
     file: restore.yml
   with_items: "{{ postgres_users }}"
-  when: item.backup_restore is defined and item.backup_restore
+  when: item.backup_restore is defined and item.backup_restore | length > 0
```

Comment:
This is a new requirement from https://docs.ansible.com/projects/ansible/latest/porting_guides/porting_guide_12.html.